feature/qos/otg_tests/sampled_output_queue_counters_test/README.md

```diff
@@ -0,0 +1,90 @@
+# DP-1.10: gNMI subscribe with sample mode for output queue counters
+
+## Summary
+WBB is required to support gNMI Subscribe with sample mode for various counters.
+This test if to verify that DUT supports gNMI Subscribe with sample mode while
+forwarding QoS traffic and updating the output queue counters correctly
+
+*   Traffic classes:
+
+    *   We will use 7 traffic classes NC1, AF4, AF3, AF2, AF1, BE0 and BE1.
+
+*   Queue types:
+
+    *   NC1 will have strict priority queues
+    *   AF4/AF3/AF2/AF1/BE1/BE0 will use WRR queues with weight 48, 12, 8, 4, 1, 1
+
+## Testbed type
+
+*   https://github.com/openconfig/featureprofiles/blob/main/topologies/atedut_2.testbed
+
+## Procedure
+
+*   Connect DUT port-1 to ATE port-1, DUT port-2 to ATE port-2
+    and configure IPv4/v6 addresses
+
+*   Configuration:
+
+    *   Configure strict priority queues for NC1.
+    *   Configure WRR for AF4, AF3, AF2, AF1, BE0 and BE1 with weight 48, 12, 8, 4, 1
+        and 1 respectively.
+
+*   Initiate traffic as per below thresholds:
+
+     Traffic class | Interface1(line rate %)
+    -------------- | -----------------------
+    NC1            | 6
+    Af4            | 48
+    AF3            | 12
+    AF2            | 8
+    AF1            | 4
+    BE1            | 1
+    BE0            | 1
+
+*   Counters should be verified using gNMI subscribe with sample mode and an interval of 10 seconds:
+*   Ensure counter of all queues increment every 10 seconds
```

Comment:
How you can be sure that DUT follow subscribe interval and not run it at "target_defined" that are by coincidence also 10sec?

Perhaps 2ns run with 20 sec interval will provide this certanity.

Reply:
We will specify "SAMPLE" as the "SubscriptionMode" and not "TARGET_DEFINED"
https://github.com/openconfig/gnmi/blob/0ac94539f877686dd812cacb66c3bfbbf307e553/proto/gnmi/gnmi.proto#L315